jobs/cloud_controller_ng/spec

```diff
@@ -1283,3 +1276,13 @@ properties:
   cc.legacy_md5_buildpack_paths_enabled:
     description: "Enable legacy MD5 buildpack paths. If disabled, xxhash64 is used for calculating paths in buildpack image layers."
     default: false
+
+  cc.temporary_enable_deprecated_thin_webserver:
```

Comment:
*nit* Perhaps we can add a note that `cc.experimental.use_puma_webserver` currently takes precedence over this. We can remove it later.